### ISSUE TITLE
gui demo for the API

### ISSUE THREAD

**Issue by astrale-sharp** [MatejSloboda/Dijkstra_map_for_Godot#4]
Hey,

I'm working on a gui demo for illustrating how the API works, if someone wants to discuss it we can do so here!

I intend to put it in a folder "API demo"
I intend to provide some godot gui to : 

buttons to illustrates function in the API:
cost_map #labels on top of the case
direction map : little arrows
connect disconnect points
connect all neighbour points
disconnect all
show point under cost
adding/show source/ target

automatic recalculate djisktramap
	recalculate for target(s) with cost

Share your thoughts !

**Comment by MatejSloboda** [comment]
I had something similar in my testing rig. I've posted images in the reddit thread, but they don't appear to work. I've managed to dig up one of them: [https://imgur.com/a/AqLzoJf](url)
Unfortunately, I'm not very familiar with Godot yet, so I just kinda hacked it in somehow with spaghetti code. I've added setter method for a dummy boolean exported variable, and the setter method recalculated the dijkstra map. I'm sure there are better ways to do it.
